rpc/chain/rpc_limiter.go

```diff
@@ -166,10 +188,21 @@ func (rl *RPCRequestLimiter) Allow(tag string) (bool, error) {
 		return true, nil
 	}
 
+	// Check if the interval has passed and reset the number of requests
+	if time.Since(data.CreatedAt) >= data.Period && data.Period.Milliseconds() != LimitInfinitely {
+		err = rl.saveToStorage(tag, data.MaxReqs, data.Period, 0, time.Now())
+		if err != nil {
+			return true, err
```

Comment:
Do we really want to return `true` in both cases?
If so then just: 
`return true, rl.saveToStorage(tag, data.MaxReqs, data.Period, 0, time.Now())`